CHANGELOG.md

```diff
@@ -0,0 +1,217 @@
+# Changelog
+
+## [Unreleased]
+
+### Added
+
+- Introduce `Radicle.toml` file for config
+- Add Dockerfile for `radicle-cli`
+- Re-organize binaries into own crate and move all sub-commands to `rad` binary 
+- Adds build metadata to version output
+- Build aarch64 Linux release binaries
+- `rad-comment`: Comment on issues or patches
+- `rad-review`: Review radicle patches
+
+### Changed
+
+- Update to replication v3
+- Update Homebrew Formula name
+- Update to rust 1.63
+- `rad-issue`: Close and re-open issues
+- `rad-patch`: Automatically push on creation, Show reviews in list, support `--message` on creation, show only local branches on list
+- `rad-rm`: Implement profile removal
+
+### Removed
+
+- `rad-auth`: Remove passphrase option, read passphrase from environment variable `RAD_PASSPHRASE` or from stdin
+
+## [0.6.1] - 2022-08-11
+
```

Comment:
@adaszko I'd need your help to sort out this section. Seems like the release artifacts are somehow out of sync. The release dates for 0.6.0 and 0.6.1 where taken from Github's release page. https://github.com/radicle-dev/radicle-cli/compare/v0.6.0...v0.6.1 only contains the release commit. Whereas https://github.com/radicle-dev/radicle-cli/compare/v0.6.1...master contains the actual commits that happened between the release dates. Could you have a look?

Reply:
There was a mess-up: 0.6.0 was not pointing to a commit in the actual history, so we had to create a new 0.6.1 tag pointing to the commit. 0.6.0 and 0.6.1 are the same code.

Reply:
Ok, I think I just found a hint in the release notes... So, I'd then keep the 0.6.1 section empty and move all the items to 0.6.0.

Reply:
Yes for 0.6.1 just say *No changes*.